ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -190,6 +190,23 @@ conformance_test(
     ],
 )
 
+conformance_test(
+    name = "conformance-test-split-participant",
+    ports = [
+        6865,
+    ],
+    server = ":app",
+    server_args = [
+        "--contract-id-seeding=testing-weak",
+        "--participant participant-id=split-example-i,port=6865,server-jdbc-url=jdbc:h2:mem:split-example;db_close_delay=-1;db_close_on_exit=false,mode=indexer",
+        "--participant participant-id=split-example-s,port=6865,server-jdbc-url=jdbc:h2:mem:split-example;db_close_delay=-1;db_close_on_exit=false,mode=ledger-api-server",
```

Comment:
The two participants can't have the same participant ID, otherwise there are conflicts in the metrics registry.